charts/cromwell/templates/_configmap.tpl

```diff
@@ -1,13 +1,30 @@
+{{- /* Generate the data component of a Cromwell config map */ -}}
+{{- define "cromwell.configmap.data" -}}
+data:
+  logback.xml: |
+{{ include "cromwell.config.logback" . | indent 4 }}
+{{- end -}}
+
 {{- /* Generate a configmap for a Cromwell deployment */ -}}
 {{- define "cromwell.configmap" -}}
 {{- $settings := ._deploymentSettings -}}
+
+{{- /*
+  Render ConfigMap data, then checksum it, and store the checksum
+  in the deploymentOutputs dictionary so that it be included as an annotation
+  on the deployment's pod template. (see _deployment.tpl)
+
+  This is used to automatically restart Cromwell pods when the ConfigMap
+  changes.
```

Comment:
Is the idea here that when the config map is updated, the checksum annotation will be updated on the deployment. This is enough to cause the controller to think the deployment has changed and will automatically perform a redeploy? If so awesome!

Reply:
Yep! Read about this technique in a random blog post somewhere, lost the link now. I definitely prefer this approach for configmap changes where contents of the config map are all deterministic / not updated by another service!